Source/Mods/RunandGun.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using Verse;
+using HarmonyLib;
+using Multiplayer.API;
+
+namespace Multiplayer.Compat
+{
+    /// <summary>RunAndGun by roolo</summary>
+    /// <see href="https://steamcommunity.com/sharedfiles/filedetails/?id=1204108550"/>
+    [MpCompatFor("roolo.RunAndGun")]
+    class RunandGun
+    {
+        Type type = AccessTools.TypeByName("RunAndGun.Harmony.Pawn_DraftController_GetGizmos_Patch");
```

Comment:
Can you make Type local?

Reply:
oh wait it is not local? give me a minute :P

Reply:
Done